collectd/build.gradle

```diff
@@ -36,15 +43,11 @@ if (!collectDLibPath.exists()) {
 archivesBaseName = 'applicationinsights-collectd'
 
 dependencies {
-    compile project(':core')
+    compile group: 'com.microsoft.azure', name: 'applicationinsights-core', version: '0.9.5'
     compile files(collectDLibPath)
     testCompile group: 'org.mockito', name: 'mockito-all', version: '1.8.0'
     testCompile group: 'junit', name: 'junit', version: '4.11'
 }
 
-// region Publishing properties
-
-projectPomName = project.msftAppInsights + " Java SDK CollectD Writer Plugin"
-projectPomDescription = "This is the CollectD writer plugin for " + project.msftAppInsightsJavaSdk
-
-// endregion Publishing properties
\ No newline at end of file
+// Building the shadow (fat) jar after compiling sources.
+tasks.assemble.doLast { tasks.shadowJar.execute() }
```

Comment:
Can't this be achieved by using dependencies which is more "Gradle-y" than calling execute()?
i.e. 

    build.dependesOn shadowJar
    shadowJar.dependesOn assemble // not sure this is needed